### ISSUE TITLE
CLI: Autocomplete database information when open

### ISSUE THREAD

**Issue by zDEFz** [keepassxreboot/keepassxc#8325]
## Summary

 When I press tab in keepassxc-cli I don't get autocompleting texts based on entry. e.g type "show -s alf" and then press tab should show results like ... alfred, alfredo... you name it)

It's easier to find content this way and usability is vastly improved

I currently use it on arch-linux

**Comment by droidmonkey** [comment]
Are you using the cli `open` command? If not then we have no idea what is actually inside your database and cannot autocomplete anything like that.

**Comment by zDEFz** [comment]
> Are you using the cli `open` command? If not then we have no idea what is actually inside your database and cannot autocomplete anything like that.

Yes I do. 

$  keepassxc-cli open /run/media/def/2TB-1/Personal/Key/DB.kdbx

Then entering the password and now I'm signed into the db.

Then I write "show -s /vim" and press {Tab} but nothing happens. That means there is a lack of auto-completion, right?
I have a entry valled "/vimeo" in the database. But it doesnt autocomplete ...

**Comment by droidmonkey** [comment]
Sure can add this as a feature for the open command

**Comment by zDEFz** [comment]
> Sure can add this as a feature for the open command

You got me wrong! This is a feature within the already opened database, not for the open command.
Applies for instance for **search**, **show**

**Comment by zDEFz** [comment]
https://user-images.githubusercontent.com/24463722/182602743-6211aff2-5d10-47f6-bc7a-06814d67ad54.mp4

**Comment by droidmonkey** [comment]
> > Sure can add this as a feature for the open command
> 
> You got me wrong! This is a feature within the already opened database, not for the open command.
> Applies for instance for **search**, **show**

I knew exactly what you intended. I meant to say this will happen AFTER a database is opened using the `open` command.

**Comment by louib** [comment]
Is that not a duplicate of https://github.com/keepassxreboot/keepassxc/issues/4676 ?

**Comment by droidmonkey** [comment]
Yes it sure is, with the caveat of also using database specific information in the auto complete, not just commands. I'll leave this open for that reason.
